source/PreferencesPanel.cpp

```diff
@@ -837,6 +857,42 @@ void PreferencesPanel::DrawPlugins()
 
 
 
+void PreferencesPanel::DrawTooltips()
+{
+	string &hoverItem = (page == 'c' ? hoverControl : hoverPreference);
+
+	// Update the tooltip timer [0-60].
+	hoverCount += !hoverItem.empty() ? (hoverCount < HOVER_TIME) : (hoverCount ? -1 : 0);
+
+	if(!hoverItem.empty() && hoverCount >= HOVER_TIME)
```

Comment:
Hmmm I feel like this would be more readable if you use guards, so like:

```c++
if(hoverItem.empty())
{
    // ...
    return;
}
// ...
if(hoverCount < HOVER_TIME)
    return;
```

Thoughts?

Reply:
I dont think thats necessary tbh

Reply:
the existing code is a bit hard to read imo, but yeah I don't mind either way